OracleDatabase/SingleInstance/FAQ.md

```diff
@@ -110,3 +114,50 @@ Refer to the [module documentation](https://python-oracledb.readthedocs.io/en/la
 ## ORA-01157: cannot identify/lock data file
 
 This error occurs when the database cannot find a data file (used for tablespaces) that was previously present. This is most likely because the data file has been located outside the volume in a previous container and was hence not persisted. Ensure that when you add tablespaces and/or data files that they are located within the volume location, i.e. $ORACLE_BASE/oradata/$ORACLE_SID, (e.g. `/opt/oracle/oradata/XE`).
+
+## Running Oracle Database 23c Free on Apple based ARM M1 chip
+
+### Setup Database
+Below are the steps to run Oracle Database 23c Free on Apple ARM M1 machine
+
+1. Install [Podman Lima](https://github.com/lima-vm/lima) on Mac
+
+    ```$ brew install podman lima```
+
+2. Start a lima instance (vm)
+
+    ```$ limactl start --name podman-amd64  --set='.arch = "x86_64" | .memory = "10GiB"'  template://podman```
+
+    Wait for the VM to boot up. Above command may error out if the VM takes a while to boot up. Ultimately it comes up fine.
+
+3. Set the address or hostname of the lima container runtime
+
+    ```$ export CONTAINER_HOST=unix://Users/$USER/.lima/podman-amd64/sock/podman.sock```
+
+4. Pull the oracle database 23c free image
+
+    ```$ podman pull container-registry.oracle.com/database/free:latest```
+
+5. Run the container with the pulled image
+
+    ```$ podman run -e ORACLE_PWD=<password> -d -P container-registry.oracle.com/database/free:latest```
+
+6. Check whether the database came up healthy.
+
+    ```sh
+    $ podman ps -a
+        CONTAINER ID  IMAGE                                               COMMAND               CREATED        STATUS                  PORTS                    NAMES 
+        825dcbd3e822  container-registry.oracle.com/database/free:latest  /bin/sh -c exec $...  4 minutes ago  Up 4 minutes (healthy)  0.0.0.0:42439->1521/tcp  hopeful_yalow
+    ```
+
+### Connect
+We need to get a Java based SQL client on Apple M1 due to unavailability of native SQL client.
```

Comment:
It's not true that you need to get a Java based SQL client.  This statement needs to be fixed before merging.

You can simply install the Intel Instant Client and Rosetta does the rest automatically when you invoke SQL*Plus.  The install instructions are the same on ARM as for Intel. You just need to install/authorize Rosetta.

Reply:
The quickest way to install SQL*Plus is via the command line:
```
cd $HOME/Downloads
curl -O https://download.oracle.com/otn_software/mac/instantclient/198000/instantclient-basic-macos.x64-19.8.0.0.0dbru.dmg
curl -O https://download.oracle.com/otn_software/mac/instantclient/198000/instantclient-sqlplus-macos.x64-19.8.0.0.0dbru.dmg
hdiutil mount instantclient-basic-macos.x64-19.8.0.0.0dbru.dmg
hdiutil mount instantclient-sqlplus-macos.x64-19.8.0.0.0dbru.dmg
/Volumes/instantclient-basic-macos.x64-19.8.0.0.0dbru/install_ic.sh
hdiutil unmount /Volumes/instantclient-basic-macos.x64-19.8.0.0.0dbru
hdiutil unmount /Volumes/instantclient-sqlplus-macos.x64-19.8.0.0.0dbru
```

Reply:
Hi @cjbj 

Updated.

Thanks